xml/bp_troubleshooting_tips.xml

```diff
@@ -782,7 +779,12 @@ ID WEIGHT TYPE NAME    UP/DOWN REWEIGHT PRIMARY-AFFINITY
   <itemizedlist mark="bullet" spacing="normal">
    <listitem>
     <para>
-     NOTE: - &cephsalt; needs to implement network diagnostics.
+     Basic network diagnostics. Try running the <literal>net.ping</literal> diagnostics
```

Comment:
how do i run the tool? is it a cephadm or salt call?...is it `salt-run network.ping` ?

Reply:
Correct. I've linked to the docs on line 786 that covers this verbosely. `net.ping` is just the name of the tool. It's all here: https://susedoc.github.io/doc-ses/master/html/ses-troubleshooting/storage-tips.html#network-test-basic-diagnostics